Probesys/bileto
# imp: Add a link to the ticket in the receipt email

## Related issue(s)



https://github.com/Probesys/bileto/issues/824

## How to test manually



1. Create an ticket
2. Verify that the sent receipt email contains a link to the ticket

## Checklist
